docs/designers-developers/developers/tutorials/javascript/readme.md

```diff
@@ -0,0 +1,108 @@
+
+# Getting Started with JavaScript
+
+The purpose of this tutorial is to step through getting started with JavaScript and WordPress. The Gutenberg documentation contains information on the APIs available, but as a reference and not necessarily full working examples. The goal of this tutorial is get you comfortable on how to use these snippets of code.
+
+
+## Plugins Background
+
+The primary means of extending WordPress remains the plugin, see [Plugin Basics](https://developer.wordpress.org/plugins/the-basics/) for more. The quick start is create a new directory in `wp-content/plugins/` to hold your plugin code, call it `myguten-plugin`.
+
+Inside of this new directory, create a file called `myguten-plugin.php` which will be the server-side code the runs when your plugin is active. For now you can place the following in that file:
+
+```php
+<?php
+/*
+Plugin Name: My Guten Plugin
+*/
+```
+
+So you should have a directory `wp-content/plugins/myguten-plugin/` which has the single file `myguten-plugin.php`. Once that is in place, you can go to your plugins list in `wp-admin` and you should see your plugin listed.
+
+Click **Activate** and your plugin will load with WordPress.
+
+
+## Loading JavaScript
+
+Now with the plugin in place, we can add our code that loads the JavaScript we want to use. This follows the standard WordPress methodology of enqueuing your scripts, see [Including CSS & JavaScript](https://developer.wordpress.org/themes/basics/including-css-javascript/) for more details.
+
+Add the following code to your `myguten-plugin.php` file:
+
+```php
+function myguten_enqueue( $hook ) {
+	wp_enqueue_script( 'myguten-script',
+		plugins_url( 'myguten.js', __FILE__ ),
+		array( 'wp-blocks', 'wp-element' )
+	);
+}
+add_action( 'admin_enqueue_scripts', 'myguten_enqueue' );
+```
+
+Create a file called `myguten.js` and add:
+
+```js
+console.log( "I'm loaded!" );
+```
```

Comment:
Probably worth recapping each of the files you have at this point.

You could also offer a direct download of this basic stub plugin to help people get going.

Reply:
I am wondering if we should use the `gutenberg-examples` repo for storing code associated with specific tutorials like this?

Reply:
The tutorial was updated and a recap added.

I like the idea of a Github repo to put the code, but the `gutenberg-examples` is so tailored for it being a plugin, it might not be the right fit. A `gutenberg-tutorials` ?  We can discuss further in #docs maybe even move content out of handbook and link to make them more stand-alone.